docs/UserGuide.md

```diff
@@ -497,7 +497,8 @@ Advanced users can edit the data file directly to modify the address book data.
 - **Error Messages**:
     - The application provides specific error messages to guide users in correcting their commands.
 - **Case Sensitivity**:
-    - Commands are generally case-insensitive, but parameters (especially for exact matches) may be case-sensitive as per the validation rules.
+  - Commands must be **lowercase** (`add, addw, edit, editw etc.`)
+  - Parameters (`n/..., e/...` ) are **case-insensitive** as per the validation rules.
```

Comment:
i think the parameters are case sensitive too
![image](https://github.com/user-attachments/assets/dce09cad-a2b5-487f-a458-a3da1227a0e2)